File: Rearrange a string - GFG/rearrange-a-string.java
// { Driver Code Starts
//Initial Template for Java

import java.io.*;
import java.util.*;

class GfG
{
    public static void main(String args[])
        {
            Scanner sc = new Scanner(System.in);
            int t = sc.nextInt();
            while(t-->0)
                {
                    String s = sc.next();
                    Solution obj = new Solution();
                    System.out.println(obj.arrangeString(s));
                }
                
        }
}
// } Driver Code Ends


//User function Template for Java

class Solution
{
    public String arrangeString(String s)
        {
            char a[]=s.toCharArray();
            java.util.Arrays.sort(a);
            StringBuffer sb=new StringBuffer();
            int sum=0;
            
            for(char i : a)
                if((int)i<=57)
                    sum+=(int)i-48;
                else
                    sb.append(i);
                    
            return sb.append(sum).toString();
        }
}

File: Rearrange a string - GFG/rearrange-a-string.py
#User function Template for python3

class Solution:
    def arrangeString(self, s):
        l=list(s);
        j=0
        for i in l:
            if i.isnumeric() : j+=int(i)
        
        return ''.join(list(sorted([i for i in l if i.isalpha()])))+str(j)

#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        s = input().strip()
        ob = Solution()
        ans = ob.arrangeString(s)
        print(ans)
# } Driver Code Ends